<launch>
    <node name="ChArUcoBoardPose" pkg="marker_localisation" type="ChArUcoBoardPoseNode.py" output="screen">
        <param name="camera_image_topic" value="/rgb/image_rect_color"/>
        <param name="camera_info_topic" value="/rgb/camera_info" />
        <param name="dictionary" value="original" />
        <param name="publish_tf" value="True" />
        <rosparam param="board_configuration">
            {
                'name':         'board1',
                'columns':      3,
                'rows':         3,
                'square_size':  0.0551,
                'marker_size':  0.0412
            }
        </rosparam>
    </node>
</launch>